test/e2e/framework/framework.go

```diff
@@ -174,30 +174,34 @@ func FleetReadyCount(amount int32) func(fleet *v1alpha1.Fleet) bool {
 	}
 }
 
-// WaitForFleetGameServersCondition wait for all GameServers for a given
-// fleet to match the spec.replicas and match a a condition
-func (f *Framework) WaitForFleetGameServersCondition(flt *v1alpha1.Fleet, cond func(server v1alpha1.GameServer) bool) error {
+// WaitForFleetGameServersCondition waits for all GameServers for a given fleet to match
+// a condition spacified by a callback.
```

Comment:
small typo
```suggestion
// a condition specified by a callback.
```

Reply:
Nice catch. Outside of this, this looks good to go :+1:

Reply:
Thanks! Fixed it.